scripts/generate-index.sh

```diff
@@ -20,4 +20,4 @@ format_as_markdown_row() {
     awk '{printf "| %s | ![%s](all-the-bufo/%s) |\n", $0, $0, $0}'
 }
 
-find_image_files | sort | extract_filename | format_as_markdown_row >> "$index_file"
```

Comment:
My system was sorting dashes after alphas, unlike the bulk of the existing file, causing a huge diff. This should force sorting consistency for everyone.